README.md

```diff
@@ -0,0 +1,287 @@
+stored in your ~/.ssh/known_hosts file. If you have modified your
+configuration file, pbuild will run this verification step again to
+double check that all your hosts can be reached. This can be forced
+with --initialize.
+
+[certificates everywhere]: https://github.com/MSFTOSSMgmt/ostc-docs/blob/master/setup-sshkeys.md
+
+2. pbuild assumes that you're using 'bash' as your shell. If important,
+it can be changed to support other shells. But to date it has no such
+support.
+
+
+### Command Qualifiers to pbuild
+
+Please do ```pbuild --help``` for latest/greatest information:
+
+```
+Usage: pbuild.py [option-list] [host-list]   (use --help or -h for help)
+
+pbuild will allow you to do a build across all platforms using your own git
+clones. If a build problem occurs, all files are left completely intact,
+simplifying the steps you must perform to replicate the problem. pbuild
+supports a wide variety of options, specified below.  For more information,
+view the README file checked in with pbuild.
+
+Options:
+  -h, --help            show this help message and exit
+  --abortOnError        Immediately aborts when the first remote build fails
+                        (for future - do not use yet)
+  --attributes=TEST_ATTRS
+                        Specifies the unit test attributes that you wish to
+                        use to restrict unit tests
+  -b BRANCH, --branch=BRANCH
+                        Selects the branch for the top level project or
+                        superproject
+  -d, --debug           Build targets in DEBUG mode
+  --clone               Forces a new clone of the repository, even if
+                        repository already exists
+  --command=COMMAND     Executes the specified command string rather than
+                        buiding/running a command script to perform a remote
+                        build
+  --exclude=EXCLUDE     Overrides default exclude list from configuration file
+                        (if any); comma-separated list of hosts to exclude
+                        from the build
+  --initialize          Verify public keys in known_hosts file
+  -l, --list            List host configuration information and exit
+  --logdir=LOGDIR       Overrides 'logdir' from configuration file
+  --logdir_prior=LOGDIR_PRIOR
+                        Overrides 'logdir_prior' from configuration file
+  --nocleanup           Skip the 'tf undo' cleanup at the end of a build
+  --nodebug             Build targets in NODEBUG mode
+  --nocurses            Disable curses for dynamic screen updating (may be
+                        useful for diagnostic purposes)
+  --select=SELECT       Select specification to build (only build hosts with
+                        this select specification)
+  --settings=SETTINGS   Overrides default settings from program and
+                        configuration file (i.e. 'ShowSummary,LogFile')
+  -s SUBPROJECT, --subproject=SUBPROJECT
```

Comment:
Added syntax example. No real point to make it a dictionary, it's just applied once, entry by entry.